lib/SILOptimizer/Utils/Generics.cpp

```diff
@@ -960,7 +906,6 @@ void ReabstractionInfo::specializeConcreteAndGenericSubstitutions(
   // Callee is the generic function being called by the apply instruction.
```

Comment:
I tried to create separate functions. But they need to share to much state. So, for now I added big comments delimiting different logical sections.